### PR TITLE
2020-05-18 GUARD-538 Product-Pull

### PR DESCRIPTION
**Pull request by fabulaspb**
Summary
Added method to pull all modified products for specified time period.

### CODE REVIEW

File: src/SkubanaAccess/Configuration/SkubanaConfig.cs [hunk 1]
```diff
@@ -33,6 +34,7 @@ public SkubanaConfig( SkubanaEnvironment environment, SkubanaUserCredentials cre
 			this.UpdateProductStockBatchSize = 200;
 			this.CreateProductStockBatchSize = 200;
 			this.RetrieveProductsBatchSize = 10;
+			this.RetrieveProductsPageSize = 20;
```
Comment (maxim-skuvault) [inline src/SkubanaAccess/Configuration/SkubanaConfig.cs:37]: Do we need both RetrieveProductsBatchSize & RetrieveProductsPageSize? Could we just have one? It's not immediately clear how they're different. I see in the solution that they're being used in different api endpoints

File: src/SkubanaAccess/Models/Product.cs [hunk 2]
```diff
@@ -20,7 +23,89 @@ public class Product
...
+				Upc = product.Upc,
+				Classifications = product.Categories,
+				Name = product.Name,
+				BrandName = product.Brand,
+				Price = product.MapPrice?.Amount ?? 0,
+				Cost = product.VendorCost?.Amount ?? 0,
+				Weight = product.Weight,
+				PartNumber = product.Mpn,
+				LongDescription = product.Description,
+				ImagesUrls = product.ImageUrls.OrderBy( i => i.Rank ).Select( i => i.Url ),
```
Comment (maxim-skuvault) [inline src/SkubanaAccess/Models/Product.cs:106]: Should we check product.ImageUrls for null?

File: src/SkubanaAccess/Configuration/SkubanaConfig.cs (hunk 1 above)
Comment (fabulaspb) [reply src/SkubanaAccess/Configuration/SkubanaConfig.cs:37]: Yes, we need both of them. Renamed RetrieveProductsBatchSize to RetrieveProductsBySkusBatchSize. It's reponsible for restricting the amount of skus that are passed to the correspondent API endpoint.

File: src/SkubanaAccess/Models/Product.cs (hunk 2 above)
Comment (fabulaspb) [reply src/SkubanaAccess/Models/Product.cs:106]: No, we don't. It's always empty array.

**Pull request merged**